src/components/Sections/SectionChart/SectionBarChart.js

```diff
@@ -161,18 +175,29 @@ const SectionBarChart = ({ data, style, dimensions, legend, chartProperties = {}
                   {...legendStyle}
                 />
               }
-              {dataItems.map(item => <Bar
-                key={item.name}
-                dataKey={item.name}
-                stackId="stack"
-                fill={item.color}
-                onClick={(e) => {
-                  if (e.url) {
-                    window.open(e.url, '_blank');
-                  }
-                }}
-                label={!!chartProperties.label}
-              />)}
+              {dataItems.map(item =>
+                <Bar
+                  key={item.name}
+                  dataKey={item.name}
+                  stackId="stack"
+                  fill={item.color}
+                  onClick={(e) => {
+                    if (e.url) {
+                      window.open(e.url, '_blank');
+                    }
+                  }}
+                  label={!!chartProperties.label}
```

Comment:
make sure this label boolean doesn't affect the LabelList child and if so, cancel it when showValues is truthy in order to maintain backwards compatibility

Reply:
for backwards compatibility 
i"m rendering the label list only in case ,chartProperties.label not exit 
![image](https://user-images.githubusercontent.com/61918543/114302654-a6f7cf00-9ad2-11eb-89b3-23753179f68c.png)